comm/client.go

```diff
@@ -1,86 +1,87 @@
 package comm
 
 import (
+	"bufio"
 	"fmt"
-	"io/ioutil"
-	"net"
+	"io"
 	"strings"
 
-	"os"
-
 	"github.com/gorilla/http"
-	//"net/http"
+	httpc "github.com/gorilla/http/client"
 
 	"github.com/phomer/scheduler/accounts"
 	"github.com/phomer/scheduler/datastore"
+	"github.com/phomer/scheduler/jobs"
 	"github.com/phomer/scheduler/log"
 )
 
 func NewClient() http.Client {
 	return http.DefaultClient
 }
 
-func MakeRequest(config *accounts.ClientConfig, request *Request) string {
+func TokenArray(config *accounts.ClientConfig) []string {
+	return []string{config.Token.Signed}
+}
+
+// Make a request to the server
+func MakeRequest(config *accounts.ClientConfig, request *jobs.Request) *Response {
```

Comment:
Instead of having separate unrelated methods for the different client-related things, I would consider collecting them into a single `Client` interface which would have methods like `Schedule`, `StreamLogs`, etc. Same on the server side.